content/docs/release/_index.md

```diff
@@ -5,6 +5,9 @@ weight: 12
 Description: >
   Dell Container Storage Modules (CSM) release notes
 ---
+{{% pageinfo color="primary" %}}
+The release 1.7.1 is only applicable to helm based installations of csi-powerflex driver.
```

Comment:
Should it be CSM 1.7.1 here?

Reply:
yes correct @rensyct 
I will update it now